client/src/components/annotators/annotator.js

```diff
@@ -72,6 +72,28 @@ export default {
     setImageCursor(newCursor) {
       this.imageCursor = `${newCursor}`;
     },
+    resetMapDimensions() {
+      this.geoViewer.bounds({
+        left: 0,
+        top: 0,
+        bottom: this.height,
+        right: this.width,
+      });
+      const params = geo.util.pixelCoordinateParams(
+        this.$refs.container,
+        this.width,
+        this.height,
+        this.width,
+        this.height,
+      );
+      this.geoViewer.maxBounds({
+        left: 0,
+        top: 0,
+        right: params.map.maxBounds.right,
+        bottom: params.map.maxBounds.bottom,
+      });
+      this.geoViewer.zoomRange(params.map);
```

Comment:
Adding this:
```
      this.geoViewer.zoom(this.geoViewer.zoomRange().origMin);
      this.geoViewer.center({ x: 0, y: 0 });
```
To here so it will recenter the image on any width/height change.  I don't know how different the image resolutions will be but it's annoying going from a 500x500 to a 2500x2500 with the same zoome level and orientation involved.  There are limits with the fact that the map has `clampBoundsX/Y` set to true to prevent you from going outside of the bounds range.